src/main/java/com/filip/workshop_managment/controllers/CustomerController.java

```diff
@@ -0,0 +1,66 @@
+package com.filip.workshop_managment.controllers;
+
+import com.filip.workshop_managment.models.Customer;
+import com.filip.workshop_managment.reopsitory.CustomerRepository;
+import com.filip.workshop_managment.reopsitory.VehicleRepository;
+import com.filip.workshop_managment.service.CustomerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class CustomerController {
+    private final CustomerRepository customerRepository;
+    private final CustomerService customerService;
+    private final VehicleRepository vehicleRepository;
+
+    @Autowired
+    public CustomerController(CustomerRepository customerRepository, CustomerService customerService,VehicleRepository vehicleRepository) {
+        this.customerRepository = customerRepository;
+        this.customerService = customerService;
+        this.vehicleRepository = vehicleRepository;
+    }
+
+    @GetMapping("/customer_list.html")
```

Comment:
mappingi zwykle tworzy się bez rozszerzeń. 
A żeby było mega koszernie - nie używa się dodatkowych określników np _list
```suggestion
    @GetMapping("/customers")
```